packages/node_modules/@ciscospark/internal-plugin-user/src/config.js

```diff
@@ -10,11 +10,11 @@ export default {
   },
 
   user: {
-    activationUrl: 'https://idbroker.webex.com/idb/token/v1/actions/UserActivation/invoke',
+    activationUrl: `${process.env.IDBROKER_BASE_URL || 'https://idbrokerbts.webex.com'}/idb/token/v1/actions/UserActivation/invoke`,
```

Comment:
We actually want to do the inverse. We will use the environment variable to create the *exception* of the BTS URL. When someone uses this library in Production, it should default to the Prod URLs. As maintainers, we will consciously override that Prod URL with the BTS URL using the environment variable.